libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -85,10 +83,33 @@ void DocStringAnalyser::handleCallable(
 	for (auto i = paramRange.first; i != paramRange.second; ++i)
 		if (!validParams.count(i->second.paramName))
 			appendError(
-				"Documented parameter \"" +
-				i->second.paramName +
-				"\" not found in the parameter list of the function."
+					"Documented parameter \"" +
+					i->second.paramName +
+					"\" not found in the parameter list of the function."
 			);
+
+}
+
+void DocStringAnalyser::handleConstructor(
+		CallableDeclaration const& _callable,
+		Documented const& _node,
+		DocumentedAnnotation& _annotation
+)
+{
+	static const set<string> validTags = set<string>{"author", "dev", "notice", "param"};
+	parseDocStrings(_node, _annotation, validTags, "constructors");
```

Comment:
Shouldn't this be `constructor` and not plural? What is the field used for?

Reply:
Hmm.. I was also thinking about that, but then I saw that within `DocStringAnalyser::handleCallable(..)` `functions` is used, thats why I also used the plural here.  It will be used to generate an error like `Error: Doc tag @return not valid for constructors.` - I had the feeling that plural is better readable.

Reply:
Looks like that you want to have the singular here.